.github/workflows/commit-issue.yml

```diff
@@ -0,0 +1,23 @@
+name: Commit Issue Workflow
+on:
+  schedule:
+    - cron: '0 * * * *' #runs every hour
+jobs:
+  commit_issue_job:
+    runs-on: ubuntu-latest
+    name: A job to create an issue if a commit was made
+    steps:
+      - uses: actions/checkout@v1
+      - name: Commit issue action step
+        id: issue
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: ./.github/actions/commit-issue-action
+        with:
+          sourceRepo: 'tree-sitter/tree-sitter-python'
+          targetRepo: 'legesher/tree-sitter-legesher-python'
+          issueLabels: 'Type: Maintenance, Status: Available, Status: Review Needed'
```

Comment:
Would you mind changing the issue label `Type: Maintenance` to be `🚧 Opportunity: Maintenance`? We recently updated our labels.

Reply:
Yup, I can do that!